config/environments/production.rb

```diff
@@ -82,6 +82,30 @@
     config.audit_logger = Logger.new(STDOUT)
   end
 
+  # Whitelist hosts
+  config.hosts << /((ci|qa)\.)?cloud\.(stage\.)?redhat\.com/
```

Comment:
`cloud` should probably be `console`. 

I wonder could this cause any issues in production behind a reverse-proxy/ingress?

Reply:
Yes it can :disappointed: